mods/screwdriver/init.lua

```diff
@@ -64,6 +64,7 @@ screwdriver.handler = function(itemstack, user, pointed_thing, mode, uses)
 		end
 	else
 		if not ndef or not ndef.paramtype2 == "facedir" or
+				ndef.on_rotate == false or
```

Comment:
I've read that accidentally the same way earlier in game_api, but that's just the return value of the `on_rotate` function itself, not the value of the `on_rotate` node-definition property.

I.e. 67: `ndef.on_rate == false` vs 60: `ndef.on_rate() == false`